index.html

```diff
@@ -200,6 +200,27 @@ <h2 class="big-text">About Myself</h2>
           <hr>
         </div>
     </section>
+    <section class="contact-section">
+      <div class="contact-Me">
+        <h3 class="contact">Contact Me</h3>
+        <p class="last-text">If you have an application you are interested in developing, a feature that you need built or a project that
+          needs coding. I’d love to help with it</p>
+        <form action="https://formspree.io/f/xvolqjkn" method="post" id="contact-form">
+          <label for="name" id="name-label"></label>
+          <input  id="name" type="text" required placeholder="Enter your Name">
+          <label for="email" id="email-label"></label>
+          <input id="email" type="email" required placeholder="example@gmail">
+          <label for="message"></label><br>
+          <textarea name="message" id="message" cols="30" rows="10" required placeholder="Write your messge here"></textarea>
+          <button type="submit" class="touch">Get in Touch</button>
```

Comment:
- [ ] Your form button design is great 👍. Can you please also follow the button styling: Active, Hover, and Pressed (see Figma design below). This makes your User Experience better.
<img width="445" alt="image" src="https://user-images.githubusercontent.com/77190203/159593366-d2fe7d79-302b-49d2-acf1-807b0b1369d0.png">